config/application.yml.example

```diff
@@ -128,6 +128,7 @@ development:
   reauthn_window: '120'
   redis_url: 'redis://localhost:6379/0'
   redis_throttle_url: 'redis://localhost:6379/1'
+  remember_device_expiration_days: '30'
```

Comment:
Should we make this required in `Figaro.rb`? I ran this locally without updating my local application.yml and saw "Remember this device for days", which sounds hip, but not specific 😄

Reply:
Yeah, I'll add it